### DISCUSSION TITLE
How to add an Outlook Category to a another users Outlook on which I have rights to do so

### DISCUSSION THREAD

**Discussion by ghost** [O365/python-o365#910] [Q&A]
Greetings,

Just had a look at the Outlook Categories documentation ( https://github.com/O365/python-o365#outlook-categories ) and adding a Category to my own Outlook works just fine.

But I have a number of 'connected' calendars from other users in which I would like to add the same category in code (when possible)

Just had a look at the code of "c:\Program Files\Python310\Lib\site-packages\O365\category.py" but cannot detect a way to make this work for subcalenders for which I am authorized.

Any suggestions on how to proceed?

**Comment by alejcas** [comment]
I don't know...

try:

```python
cat = account.outlook_categories(resource='other_user_with_connected_calendar')
```
